tests/test_wave_01.py

```diff
@@ -159,12 +159,15 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    #raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
-
-@pytest.mark.skip()
+    assert updated_data["watched"][-1]["title"] == MOVIE_TITLE_1
+    assert updated_data["watched"][-1]["genre"] == GENRE_1
+    assert updated_data["watched"][-1]["rating"] == RATING_1
```

Comment:
Nice! This could look like this as well: 
```python 
assert updated_data["watched"][-1] == {
            "title": MOVIE_TITLE_1,
            "genre": GENRE_1,
            "rating": RATING_1
        }
```